modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -624,4 +633,44 @@ private void addEncoderAndDecoder(Map<String, Object> vendorExtensions, String d
         ELM_018,
         ELM_019
     }
+
+    @Override
+    public void postProcessFile(File file, String fileType) {
+        if (file == null) {
+            return;
+        }
+
+        String elmFmtPath = System.getenv("ELM_FORMAT_PATH");
+        if (StringUtils.isEmpty(elmFmtPath)) {
+            return; // skip if ELM_FORMAT_PATH env variable is not defined
+        }
+
+        // only procees the following type (or we can simply rely on the file extension to check if it's a elm file)
```

Comment:
I'd skip this part and just check for `.elm`

Reply:
Sure. I can remove that, which should have no impact on the result.